store/tikv/errno/errcode.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2020 PingCAP, Inc.
```

Comment:
2021？

Reply:
got it, modified the year in last commit